org-formation/005-types/_tasks.yaml

```diff
@@ -0,0 +1,114 @@
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Region: us-east-1 # Only compatible to us-east-1 region
+
+AccountPublicAccessBlockRp:
+  Type: register-type
+  ResourceType: 'Community::S3::PublicAccessBlock'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-s3-publicaccessblock-0.2.0.zip'
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion # Public Access Block is an account wide setting. This only needs to be installed to the primary region
+
+OrganizationsDelegatedAdminRp:
+  Type: register-type
+  ResourceType: 'Community::Organizations::DelegatedAdmin'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-organizations-delegatedadmin-0.1.0.zip'
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Region: us-east-1 # Only compatible to us-east-1 region
+
+EnableAWSServiceRp:
+  Type: register-type
+  ResourceType: 'Community::Organizations::EnableAWSServiceAccess'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-organizations-enableawsserviceaccess-0.1.0.zip'
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Region: us-east-1 # Only compatible to us-east-1 region
+
+PasswordPolicyRp:
+  Type: register-type
+  ResourceType: 'Community::IAM::PasswordPolicy'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-iam-passwordpolicy-0.2.0.zip'
+  MaxConcurrentTasks: 10
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion
+
+ServiceQuotasS3Rp:
+  Type: register-type
+  ResourceType: 'Community::ServiceQuotas::S3'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-servicequotas-s3-0.1.0.zip'
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref allRegions
+
+ServiceQuotasCloudFormationRp:
+  Type: register-type
+  ResourceType: 'Community::ServiceQuotas::CloudFormation'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-servicequotas-cloudformation-0.1.0.zip'
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref allRegions
+
+ServiceQuotasDynamoDbRp:
+  Type: register-type
+  ResourceType: 'Community::ServiceQuotas::DynamoDB'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-servicequotas-dynamodb-0.1.0.zip'
+  MaxConcurrentTasks: 100
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref allRegions
+
+SsoAssignmentGroupRp:
+  Type: register-type
+  ResourceType: 'Community::SSO::AssignmentGroup'
+  SchemaHandlerPackage: !Sub 's3://${catalogBucket}/community-sso-assignmentgroup-0.3.1.zip'
+  MaxConcurrentTasks: 10
+  OrganizationBinding:
+    IncludeMasterAccount: true
+    Region: !Ref primaryRegion
```

Comment:
Same as above... why only primary?

Reply:
i just copied these settings from the ofn reference project.  I currently have primaryRegion and all AllRegions both set to us-east-1 right now.  so it's the same anyways